handler/setup.go

```diff
@@ -200,6 +202,22 @@ func getDockerSockVolume() *spec.Volume {
 	}
 }
 
+func appendConfigVolume(volumes []*spec.Volume) []*spec.Volume {
+	loadedConfig, _ := config.Load()
+	droneRunnerVolumes := loadedConfig.DRONE_RUNNER_VOLUMES
+	for _, droneRunnerVolume := range droneRunnerVolumes {
+		volume := &spec.Volume{
+			HostPath: &spec.VolumeHostPath{
+				Name: strings.Split(droneRunnerVolume, ":")[0],
```

Comment:
Create another function to parse the volume so the logic is easy to see - similar to https://github.com/drone-runners/drone-runner-aws/pull/88/files#diff-3c718f8d6204bf9137ea1ed7872d05de927fb5ef0d8d082f4fe5df62cd0c3600R11